tests/e2e/instrumentation-dotnet-multicontainer/00-install-instrumentation.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: dotnet
+spec:
+  env:
+    - name: OTEL_TRACES_EXPORTER
+      value: otlp
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: http://localhost:4317
+    - name: OTEL_EXPORTER_OTLP_TIMEOUT
+      value: "20"
+    - name: OTEL_TRACES_SAMPLER
+      value: parentbased_traceidratio
+    - name: OTEL_TRACES_SAMPLER_ARG
+      value: "0.85"
+    - name: SPLUNK_TRACE_RESPONSE_HEADER_ENABLED
+      value: "true"
+  exporter:
+    endpoint: http://localhost:4317
+  propagators:
+    - jaeger
+    - b3
+  sampler:
+    type: parentbased_traceidratio
+    argument: "0.25"
+  dotnet:
+    env:
+    - name: OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS
```

Comment:
is this required? What happens if the variable is empty?

Reply:
The PR [#932 Enable all trace instrumentations by default (simplified)](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/932) for enabling the all trace instrumentation by default was merged few days back. 

So, `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` is required to be specified for the release [opentelemetry-dotnet-instrumentation v0.2.0-beta.1](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v0.2.0-beta.1).

If it is empty, then applications runs but trace instrumentation does not work.

Reply:
Will this change in the future? If no the operator should set this variable on the behalf of the user.

Reply:
Previously, this env variable's name was `OTEL_DOTNET_TRACER_INSTRUMENTATIONS `. In v0.2.0-beta.1, it is renamed to `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS `. 

As of now, we can set this env value to `AspNet,HttpClient,SqlClient` but in the next release of the dotnet auto-instrumentation `Npgsql, MySqlData` will be added.

Reply:
We should definitely document this in the readme and the operator could set the variable (if it is not set) in the defaulting webhook

Reply:
Done. I have pushed the changes.